ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Conversions.scala

```diff
@@ -215,4 +217,20 @@ private[state] object Conversions {
       )
     )
 
+  def encodeBlindingInfo(blindingInfo: BlindingInfo): DamlTransactionBlindingInfo = {
```

Comment:
We don't seem to have unit tests for conversions although we probably should, shouldn't we? I'll look around once more and I'll create add unit test for this one.

Reply:
Introduced.